tox.ini

```diff
@@ -125,10 +126,30 @@ deps =
 commands =
     mkdir -p .report
     coverage run --source={[vars]src_path},testing/src/scenario \
-             -m pytest --ignore={[vars]tst_path}smoke -v --tb native {posargs}
+             -m pytest --ignore={[vars]tst_path}smoke --ignore={[vars]tst_path}test_benchmark.py \
+             -v --tb native \
+             -W 'ignore:Harness is deprecated:PendingDeprecationWarning' {posargs}
     coverage xml -o .report/coverage.xml
     coverage report
 
+[testenv:benchmark]
+description = Run benchmark tests
+passenv =
+    RUN_REAL_PEBBLE_TESTS
+    PEBBLE
+deps =
+    PyYAML==6.*
+    websocket-client==1.*
+    coverage[toml]~=7.0
```

Comment:
Can we drop coverage for running benchmarks? In fact, could this slow things down / mess with the results?

Reply:
Ah, good catch, thanks. I had lazily just copied the dependencies from the coverage environment. It shouldn't change the results because I'm not actually running `coverage` but it's indeed unnecessary.